unstructured/documents/elements.py

```diff
@@ -12,7 +12,7 @@
 import re
 import uuid
 from types import MappingProxyType
-from typing import Any, Callable, FrozenSet, Optional, Sequence, cast
+from typing import Any, Callable, FrozenSet, List, Optional, Sequence, cast
```

Comment:
Use lowercase `list` instead of importing and using `Typing.List`. As long as `from __future__ import annotations` appears at the top this will work fine and is Python-version forward-compatible. Occasionally you will need to place the declaration in quotes, like maybe for a `cast()` call or `TypeAlias` instance. The type-checker will tell you where that is necessary but it is infrequent.

Reply:
![image](https://github.com/Unstructured-IO/unstructured/assets/64484917/6d0bd619-4f25-48ed-919b-626d5f153745)
done